src/dotnet/APIView/APIViewWeb/Managers/Interfaces/IAPIRevisionsManager.cs

```diff
@@ -41,5 +41,6 @@ public interface IAPIRevisionsManager
         public Task AssignReviewersToAPIRevisionAsync(ClaimsPrincipal User, string apiRevisionId, HashSet<string> reviewers);
         public Task<IEnumerable<APIRevisionListItemModel>> GetAPIRevisionsAssignedToUser(string userName);
         public Task<APIRevisionListItemModel> UpdateRevisionMetadataAsync(APIRevisionListItemModel revision, string packageVersion, string label);
+        public Task<APIRevisionListItemModel> GetRevisionForPackageVersionAsync(string reviewId, string packageVersion, APIRevisionType apiRevisionType);
```

Comment:
Perhaps you could extend `GetAPIRevisionsAsync(string reviewId)` such that `packageVersion` and `apiRevisionType` are optional parameters.

Reply:
done